### PR TITLE
Style import rework

### PR DESCRIPTION
**Pull request by SKProCH**
Add `MaterialTheme` instead of separate StyleInclude and BundledTheme importing.

`MaterialTheme` is a bootstrap for quick applying color to the application. If user need more control over the colors - then he should use MaterialThemeBase and set custom theme by hands.

Also this PR removes all obsolete trash that comes from early releases of MDiX, like xaml files with colors, `Swatch` and `Hue` classes. Commits: de16fbf5 and af08a096. 

New MaterialTheme theming are incompatible with old system, but i think this dont break existing apps.

### CODE REVIEW

**Review by CreateLab — APPROVED** [review]

File: Material.Styles/Themes/MaterialThemeBase.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        private bool _isLoading;
+        private IStyle? _loaded;
+        
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FluentTheme"/> class.
+        /// </summary>
+        /// <param name="baseUri">The base URL for the XAML context.</param>
+        public MaterialThemeBase(Uri baseUri) {
+            _controlsStyles = new StyleInclude(baseUri) {
+                Source = new Uri("avares://Material.Avalonia/Material.Avalonia.Templates.xaml")
```
Comment (appleneko2001) [inline Material.Styles/Themes/MaterialThemeBase.cs:23]: Recommended TODO: resources include index integration (without Material.Avalonia.Templates.xaml)

**Comment by Al-Dyachkov** [comment]
What should be used instead of "ValidationErrorColor" ?
Just pick any red ?

**Comment by appleneko2001** [comment]
Hi @Al-Dyachkov !

> What should be used instead of "ValidationErrorColor" ? Just pick any red ?

You can use `ValidationErrorBrushColor` instead of `ValidationErrorColor`

**Review by appleneko2001 — COMMENTED** [review]
Might crash when applying themes asynchronously.
Anyway I will approve those changes and fix them later.

**Pull request merged**